README.md

```diff
@@ -10,7 +10,7 @@ module "postgresql" {
 
   product   = var.product
   component = var.component
```

Comment:
can you reformat these so it lines up with business area? otherwise people will have to run `terraform fmt` when they copy paste

Reply:
I thought I ran that, I have just run that again but it did not update anything